flink-filesystems/flink-s3-fs-presto/src/main/resources/META-INF/NOTICE

```diff
@@ -34,14 +34,25 @@ This project bundles the following dependencies under the Apache Software Licens
 - joda-time:joda-time:2.5
 - org.apache.commons:commons-configuration2:2.1.1
 - org.apache.commons:commons-lang3:3.3.2
-- org.apache.hadoop:hadoop-annotations:3.1.0
-- org.apache.hadoop:hadoop-aws:3.1.0
-- org.apache.hadoop:hadoop-auth:3.1.0
-- org.apache.hadoop:hadoop-common:3.1.0
+- org.apache.commons:commons-text:1.4
+- org.apache.hadoop:hadoop-annotations:3.3.1
+- org.apache.hadoop:hadoop-aws:3.3.1
+- org.apache.hadoop:hadoop-auth:3.3.1
+- org.apache.hadoop:hadoop-common:3.3.1
+- org.apache.hadoop.thirdparty:hadoop-shaded-guava:1.1.1
+- org.apache.hadoop.thirdparty:hadoop-shaded-protobuf_3_7:1.1.1
 - org.apache.htrace:htrace-core4:4.1.0-incubating
-- org.apache.httpcomponents:httpcore:4.4.14
 - org.apache.httpcomponents:httpclient:4.5.13
+- org.apache.httpcomponents:httpcore:4.4.14
+- org.apache.kerby:kerby-asn1:1.0.1
+- org.apache.kerby:kerb-core:1.0.1
+- org.apache.kerby:kerby-pkix:1.0.1
+- org.apache.kerby:kerby-util:1.0.1
+- org.codehaus.woodstox:stax2-api:4.2.1
+- org.xerial.snappy:snappy-java:1.1.8.3
 - org.weakref:jmxutils:1.19
+- org.wildfly.openssl:wildfly-openssl:1.0.7.Final
+- dnsjava:dnsjava:2.1.7
```

Comment:
Did you verify that this is indeed ASL?

Reply:
@AHeise after a quick check, these are the license for the two dependencies you have highlighted

DNSJava 2.1.7 is licensed under BSD-2 Clause
https://mvnrepository.com/artifact/dnsjava/dnsjava/2.1.7


WildFly OpenSSL is licensed under Apache 2.0 
https://mvnrepository.com/artifact/org.wildfly.openssl/wildfly-openssl/1.0.7.Final

/cc @srinipunuru

Reply:
So `dnsjava` is definitively declared incorrectly then. Thanks for checking!

Reply:
Thanks @AHeise , that's a miss. I will fix this and also verify the other ones as well.

Reply:
Thanks for this comment, I did a scan of all the libraries, I fixed the licenses for all the libraries that are introduced because of the hadoop upgrade.